### DISCUSSION TITLE
React Native + Dynamic Links with params

### DISCUSSION THREAD

**Discussion by pupato13** [invertase/react-native-firebase#5024] [Q&A]
Hi guys, I'm developing a React Native APP, and I'm trying to create a URL with firebase dynamic links and set a parameter to fetch data accordingly to that param.

For example, I have my Dynamic Link configured as "https://myApp.page.link/my-satellite".

If I type on the browser "https://myApp.page.link/my-satellite", my App opens and go to that Screen.

What I need is to pass "https://myApp.page.link/my-satellite/SERIAL_CODE_PARAM" and fetch data from SERIAL_CODE_PARAM, but the Firebase console doesn't allow me to create that URL.

Could someone please, help me develop that?

Cheers

**Comment by mikehardy** [comment]
This is not react-native-firebase specific, I don't anticipate help coming here. You should post this on stackoverflow or similar to figure out how to pass parameters on dynamic links. In general I think you encode the parameters into the link, which means you will want to generate the links programmatically

**Reply by pupato13** [reply]
Hey @mikehardy, thanks for your reply.

I've posted a question asking that on Stackoverflow before come here!

Also, I followed firebase documentation on how to create dynamic links manually, the problem is my app didn't trigger any event. That's why I asked for help here if someone has implemented it and could help me!

Cheers

**Comment by pupato13** [comment]
Hey @mikehardy, you can close this discussion.

Cheers

**Reply by mikehardy** [reply]
DIscussions stay forever. If you have a solution you should post the solution, then you can mark it as an answer, that's how discussions are "closed"
